webportal/lambda/.chalice/config.json

```diff
@@ -94,18 +94,18 @@
         "TABLENAME_USER_STACKS": "prod-UserStacksTable",
         "TABLENAME_AVAILABLE_DATASET": "prod-AvailableDataset",
         "TABLENAME_AUTOEXPORT_USERS": "prod-AutoExportUsersTable",
-        "RESTAPIID": "3lcauverj5",
-        "AUTHORIZERID": "04vmys",
-        "TABLENAME_TRUSTED_USERS": "prod-TrustedUsersTable",
-        "TABLENAME_EXPORT_FILE_REQUEST": "prod-RequestExportTable",
-        "TABLENAME_MANAGE_USER": "prod-ManageUserWorkstationTable",
+        "RESTAPIID": "3jca2s255l",
+        "AUTHORIZERID": "tvt9se",
+        "TABLENAME_TRUSTED_USERS": "production-TrustedUsersTable",
+        "TABLENAME_EXPORT_FILE_REQUEST": "production-RequestExportTable",
+        "TABLENAME_MANAGE_USER": "production-ManageUserWorkstationTable",
         "TABLENAME_MANAGE_USER_INDEX": "prod-workstation-username-index",
         "TABLENAME_MANAGE_DISK": "prod-ManageDiskspaceRequestsTable",
         "TABLENAME_MANAGE_DISK_INDEX": "prod-diskspace-username-index",
-        "TABLENAME_MANAGE_UPTIME": "prod-ScheduleUptimeTable",
+        "TABLENAME_MANAGE_UPTIME": "production-ScheduleUptimeTable",
```

Comment:
We bouncing between `prod` and `production` again?

Reply:
Yes :-(